app/src/main/java/com/stevesoltys/seedvault/ui/notification/BackupNotificationManager.kt

```diff
@@ -16,6 +16,7 @@ import androidx.core.app.NotificationCompat.Builder
 import androidx.core.app.NotificationCompat.PRIORITY_DEFAULT
 import androidx.core.app.NotificationCompat.PRIORITY_HIGH
 import androidx.core.app.NotificationCompat.PRIORITY_LOW
+import com.stevesoltys.seedvault.BuildConfig.APPLICATION_ID
```

Comment:
This won't be available when building in AOSP, without gradle.

Looks like it's already in the other notification commit too - https://github.com/stevesoltys/seedvault/pull/111 - that'll break build.